source/documentation/managing_apps/autoscaling.erb

```diff
@@ -0,0 +1,278 @@
+## Autoscaling
+
+Apps can be configured to dynamically scale in and out based on autoscaling
+policies.
+
+Scaling your app to fewer instances can reduce the cost during periods of low
+usage,
+eg at night.
+
+Scaling your app based on throughput or average latency can improve the user
+experience of your app during periods of high traffic,
+eg before a submission deadline.
+
+### Scaling methods
+
+There are two ways of scaling:
+
+1. `schedules` - scale apps based on time of day and day of week or month
+1. `scaling_rules` - scale apps based on metrics, eg average latency
+
+Schedules and scaling rules can be used at the same time, or on their own.
+
+### Limits
+
+There are two limits which you can apply to an autoscaling policy:
+
+* `instance_max_count` - maximum number of instances to run
+* `instance_min_count` - minimum number of instances to run
```

Comment:
I assume `instance_min_count` must be at least `1` (or there would be no metrics to base scaling on right?) ... is it worth calling out the value should be `>0` ... or note elsewhere that this is not about `scale-to-zero` type processes. Unless I'm wrong and you can scale to zero using `responsetime` / `throughput` metrics somehow.

Reply:
I was hoping that the minimum would be zero, which would `Turn off at night` autoscaling policies, however the minimum is 1.

A user gets a descriptive error message:

```
Error: Invalid policy definition:
(root).instance_min_count: Must be greater than or equal to 1.
```

Reply:
Addressed in e0765be